index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">  
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">  
+    <title>Survey Form</title>
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action=" hello" id="survey-form">
```

Comment:
It would be nice if you try to space-less inside your quotes

action=" hello", so, can you delete empty space before hello?. 🤓

Reply:
- [ ] Great observation and suggestion. But as I have mentioned above this is part of the HTML&CSS requirements so it should be required and implemented by the student. So let's ask the student in an elegant way to make the changes.